third_party/move/move-examples/diem-framework/crates/natives/Cargo.toml

```diff
@@ -7,11 +7,9 @@ edition = "2021"
 license = "Apache-2.0"
 
 [dependencies]
-move-binary-format = { path = "../../../../../move-binary-format" }
-move-core-types = { path = "../../../../../move-core/types" }
-move-vm-runtime = { path = "../../../../../move-vm/runtime" }
-move-vm-types = { path = "../../../../../move-vm/types" }
-
 diem-crypto = { path = "../crypto" }
```

Comment:
Could be made `{ workspace = true }`?

Reply:
For this one we probably do not even need natives or crates, as far as I understand we use it for prover testing. @junkil-park @rahxephon89 we just use the Move code, right?

Reply:
So my idea was not to add this to the workspace path for that reason.

Reply:
Yeah, prover does not use natives